src/modules/Relayer.ts

```diff
@@ -0,0 +1,113 @@
+/*
+
+    Copyright 2020 dYdX Trading Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+*/
+
+import BigNumber from 'bignumber.js';
+import { Contracts } from '../modules/Contracts';
+import {
+  CallOptions,
+  SendOptions,
+  MakerOracleMessage,
+} from '../lib/types';
+import { signatureToVRS } from '../lib/SignatureHelper';
+
+export class Relayer {
```

Comment:
a brief description e.g. `Used by authorized price relayers to call functions on the Maker oracle contract.`